### ISSUE TITLE
Conflicts with mini_magick (~> 4.5.1) (Gem::ConflictError)

### ISSUE THREAD

**Issue by ricardopereira** [HazAT/badge#56]
It only happens while using `badge`. I don't know if this is a problem with `badge`. If not then please say so 😅

```
> badge --dark
/Users/ricardopereira/.rbenv/versions/2.2.3/lib/ruby/site_ruby/2.2.0/rubygems/specification.rb:2288:in `raise_if_conflicts': Unable to activate fastlane-2.50.1, because mini_magick-4.8.0 conflicts with mini_magick (~> 4.5.1) (Gem::ConflictError)
	from /Users/ricardopereira/.rbenv/versions/2.2.3/lib/ruby/site_ruby/2.2.0/rubygems/specification.rb:1408:in `activate'
	from /Users/ricardopereira/.rbenv/versions/2.2.3/lib/ruby/site_ruby/2.2.0/rubygems.rb:208:in `rescue in try_activate'
	from /Users/ricardopereira/.rbenv/versions/2.2.3/lib/ruby/site_ruby/2.2.0/rubygems.rb:201:in `try_activate'
	from /Users/ricardopereira/.rbenv/versions/2.2.3/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:125:in `rescue in require'
	from /Users/ricardopereira/.rbenv/versions/2.2.3/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:40:in `require'
	from /Users/ricardopereira/.gem/ruby/2.2.0/gems/badge-0.8.4/lib/badge/options.rb:1:in `<top (required)>'
	from /Users/ricardopereira/.rbenv/versions/2.2.3/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/ricardopereira/.rbenv/versions/2.2.3/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/ricardopereira/.gem/ruby/2.2.0/gems/badge-0.8.4/lib/badge.rb:3:in `<top (required)>'
	from /Users/ricardopereira/.rbenv/versions/2.2.3/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/ricardopereira/.rbenv/versions/2.2.3/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/ricardopereira/.gem/ruby/2.2.0/gems/badge-0.8.4/bin/badge:4:in `<top (required)>'
	from /Users/ricardopereira/.gem/ruby/2.2.0/bin/badge:22:in `load'
	from /Users/ricardopereira/.gem/ruby/2.2.0/bin/badge:22:in `<main>'
```

I uninstalled the mini_magick v4.8.0 and now it works.

```
> gem uninstall mini_magick --version 4.8.0
Successfully uninstalled mini_magick-4.8.0
```

Does `badge` need any sort of dependencies update?

**Comment by sergii-frost** [comment]
I agree. There are some issues with mini_magick 4.5.x - if fails quite often when validating image.

The latest version available is `4.8.0` but I can't update to this version, getting error like:

```
Bundler could not find compatible versions for gem "mini_magick":
  In Gemfile:
    mini_magick (= 4.8.0)

    badge (= 0.8.4) was resolved to 0.8.4, which depends on
      mini_magick (>= 4.5)

    badge (= 0.8.4) was resolved to 0.8.4, which depends on
      fastlane (>= 2.0) was resolved to 2.47.0, which depends on
        mini_magick (~> 4.5.1)
```

**Comment by HazAT** [comment]
Should be fixed in `0.11.0`
